web/tests/unit/example.spec.ts

```diff
@@ -0,0 +1,5 @@
+describe('Component under test', () => {
```

Comment:
Do we need this example test file at all? I can't tell what it does other than act as a template.

Reply:
I think it's just there to fill out the testing infrastructure until actual tests are needed. It will go away when real tests arrive, and as such I think we do need it until then.

Reply:
valid enough reason to keep it :+1: